boards/arm64/xenvm/xenvm_defconfig

```diff
@@ -9,6 +9,7 @@ CONFIG_FLASH_BASE_ADDRESS=0x0
 CONFIG_FLASH_SIZE=0
 CONFIG_AARCH64_IMAGE_HEADER=y
 CONFIG_MAX_XLAT_TABLES=10
+CONFIG_HEAP_MEM_POOL_SIZE=16384
```

Comment:
Can you clarify why this is necessary and how this was decided?

Reply:
By default Zephyr has not memory at heap memory pool, so you can not use dynamic allocation at all.
This defconfig is for Xen VM sample build and now, when grant tables are introduced, it is needed to have some memory for alloc_and_grant testing. This value is equal to 4 pages, that was enough for development testing. Also it can be changed easily when someone decides to build something more difficult on xenvm sample.